Дз4/css/style_for_display.css

```diff
@@ -0,0 +1,119 @@
+div {
+  text-align: center;
+  display: inline-block;
```

Comment:
display: inline-block; and display: flex; is different solutions for different tasks